src/main/java/io/xcherry/core/worker/WorkerService.java

```diff
@@ -79,13 +109,13 @@ public AsyncStateExecuteResponse handleAsyncStateExecute(final AsyncStateExecute
 
         return new AsyncStateExecuteResponse()
             .stateDecision(toApiModel(request.getProcessType(), stateDecision))
-            .publishToLocalQueue(communication.getLocalQueueMessagesToPublish())
-            .writeToGlobalAttributes(
-                persistence.getGlobalAttributesToUpsert(registry.getPersistenceSchema(request.getProcessType()))
-            );
+            .publishToLocalQueue(communication.getLocalQueueMessagesToPublish());
+        //            .writeToGlobalAttributes(
```

Comment:
nit: feel free to remove them if they are not needed , otherwise leave a comment for how to uncomment it back later?

Reply:
Once the database-related logic has been implemented in the server, I will modify these codes here accordingly.